src/data/list/big_operators.lean

```diff
@@ -0,0 +1,403 @@
+/-
+Copyright (c) 2014 Parikshit Khanna. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Parikshit Khanna, Jeremy Avigad, Leonardo de Moura, Floris van Doorn, Mario Carneiro
```

Comment:
I'm going to leave you to do the author sleuthing on this one :)

Reply:
* Johannes for b031441fa4b7ca428ca7dd3267a4cb9472fab284.
* Mario for f9de18397dc1a43817803c2fe5d84b282287ed0d.
* Chris for #272, #1679.
* Floris for #1466.
* Sébastien for #1820.
* Yury for #1820.
* Scott for #2342, #2345, #2686.
* Kevin for #2385.
* Johan for #2686.
* Bryan for #2686.

and I'm probably missing some. Heeeelp.

Reply:
It looks like all I did in #2686 was push a merge commit, so you definitely don't need to put me in the authors for that.